cli/waiter/display.py

```diff
@@ -0,0 +1,124 @@
+import collections
+
+from tabulate import tabulate
+
+from waiter import terminal
+from waiter.format import format_last_request_time, format_mem_field, format_memory_amount, format_status
+from waiter.util import is_service_current, print_error
+
+
+def retrieve_num_instances(service):
+    """Returns the total number of instances."""
+    instance_counts = service["instance-counts"]
+    return instance_counts["healthy-instances"] + instance_counts["unhealthy-instances"]
+
+
+def format_using_current_token(service, token_etag, token_name):
+    """Formats the "Current?" column for the given service"""
+    is_current = is_service_current(service, token_etag, token_name)
+    if is_current:
+        return terminal.success('Current')
+    else:
+        return 'Not Current'
+
+
+def format_instance_healthy(instance):
+    """Formats the "Healthy?" column for the given instance"""
+    return terminal.success('Healthy') if instance['healthy?'] else terminal.failed('Unhealthy')
```

Comment:
We spoke about adding the `Failed` and `Killed` annotations on service instances.